tests/acceptance/manager/BillBulkButtonCest.php

```diff
@@ -0,0 +1,106 @@
+<?php
+declare(strict_types=1);
+
+namespace hipanel\modules\finance\tests\acceptance\manager;
+
+use hipanel\helpers\Url;
+use Codeception\Example;
+use hipanel\tests\_support\Page\IndexPage;
+use hipanel\tests\_support\Step\Acceptance\Manager;
+use hipanel\modules\finance\tests\_support\Page\bill\Create;
+
+class BillBulkButtonCest
+{
+    /**
+     * @dataProvider BillDataProvider
+     */
+    public function ensureBulkButtonsWorkCorrectly(Manager $I, Example $example): void
+    {
+        $I->login();
+        $example = iterator_to_array($example->getIterator());
+        $I->needPage(Url::to('@bill/index/?sort=-time'));
+
+        $this->checkRows($I, [1, 2]);
+        $I->pressButton($example['action']);
+
+        $this->{$example['method']}($I, $example['descrp']);
+    }
+
+    private function checkRows(Manager $I, array $rows): void
+    {
+        foreach ($rows as $row) {
+            $I->checkOption("//tbody//tr[$row]//input");
+        }
+    }
```

Comment:
```suggestion
         $I->checkOption("//tbody//tr[0]//input");
         $I->checkOption("//tbody//tr[1]//input");
```
забагато кода, щоб перевірить 2 опшна